app/models/item.rb

```diff
@@ -34,6 +34,7 @@ class Item < ApplicationRecord
   scope :alphabetized, -> { order(:name) }
   scope :by_base_item, ->(base_item) { where(base_item: base_item) }
   scope :by_partner_key, ->(partner_key) { where(partner_key: partner_key) }
+  scope :include_inactive_items, ->(_) { where(active: [true, false]) }
```

Comment:
This is dangerous because true/false is in sql land, and it may be null, in which case it's neither. We're killing this scope anyways.